README.md

```diff
@@ -159,6 +166,20 @@ $response = $matej->request()
     ->send();
 ```
 
+`send()` method returns a Recommendations response, which will allows you to access full response data:
```

Comment:
Not sure here, but shouldn't it be "which will allow you"?

Reply:
And maybe "which will allow you to directly access response data:" (to emphasize the difference from using `getCommandResponse()` / `getCommandResponses()`)

(Same for the Sorting request below)